src/components/home/TimeTracker.js

```diff
@@ -32,11 +32,33 @@ export const GrowthHours = ({ hoursUsed, hoursRemaining, totalGrowth }) => {
 }
 
 const TimeTracker = () => {
-  const { billable, growth } = EmployeeTimeTracking
+  const initialState = {
+    billable: {
+      currentHours: 0,
+      currentTarget: 0,
+      totalTarget: 0,
+    },
+    growth: {
+      hoursUsed: 0,
+      hoursRemaining: 0,
+      totalGrowth: 0,
+    },
+  }
+
+  const [data, setData] = useState(initialState)
+
+  useEffect(() => {
+    API.get("http://localhost:5000/Employee/")
+      .then(res => {
+        setData(res.data)
+      })
+      .catch(ex => console.log(ex))
```

Comment:
There's a chance we want to make this call from another component. We should extra this out into some file (/src/api/index.js???) so we have the potential to import it elsewhere.